src/features/class-static-initializer-blocks.md

```diff
@@ -0,0 +1,108 @@
+---
+title: 'Class static initializer blocks'
+author: 'Shu-yu Guo ([@_shu](https://twitter.com/_shu))'
+avatars:
+  - 'shu-yu-guo'
+date: 2021-03-19
+tags:
+  - ECMAScript
+description: 'JavaScript classes get dedicated syntax for static initialization.'
+---
+The new class static initializer block syntax lets developers gather code that should run once for a given class definition and put them in a single place. Consider the following example where a pseudo random number generator uses a static block to initialize an entropy pool once, when the `class MyPRNG` definition is evaluated.
+
+```js
+class MyPRNG {
+  seed;
```

Comment:
This seems unnecessary?

Reply:
The declaration? Eh, I think some folks like declaring the fields they're going to use up front.

Reply:
I think this  is common, especially with TS requiring you to forward declare.

Reply:
@syg I think I'd still remove this - it's rather unusual to see in pure JS for regular properties, and you did say below you want to minimize number of new features to avoid distractions.